.angular-cli.json

```diff
@@ -16,6 +16,11 @@
             "testTsconfig": "tsconfig.spec.json",
             "prefix": "app",
             "styles": ["styles.scss"],
+            "stylePreprocessorOptions": {
```

Comment:
This it's what allows me to do this: https://github.com/dotCMS/core-web/pull/419/files#diff-a09f6a8154f077be31dd14760c18666aR1

Reply:
cool